### PR TITLE
peewee_migrate migrations support

### PR DESCRIPTION
**Pull request by nauar**
Solves issue #32 

No need for setupdb.py anymore. It loads all the migrations on startup.

### CODE REVIEW

File: migrations/002_migrations.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+def migrate(migrator: Migrator, database: pw.Database, *, fake=False):
+  if Path("gametypes.json").exists():
+    try:
+        f = open("gametypes.json", encoding="utf-8")
+        gametypes = json.loads(f.read()).items()
+        f.close()
+        for gametitle, gameinfo in gametypes:
+            GameTypes.get_or_create(title=gametitle, playerCount=gameinfo["playerCount"], teamCount=gameinfo["teamCount"], statsName=gameinfo["statsName"])
+    except Exception as e:
+        print('Error: %s', repr(e)) #("No table for gametypes found! Please execute 'createdb' first.")
```
Comment (Seekfried) [inline migrations/002_migrations.py:20]:
```suggestion
        print('Error: %s', repr(e))
```
setupdb.py is not there anymore

File: pickups_old.db
Comment (Seekfried) [inline pickups_old.db:1]: this is just a backup? shouldnt be commited

**Review by Seekfried — CHANGES_REQUESTED** [review]

File: pickups_old.db
Comment (nauar) [reply pickups_old.db:1]: Ooops! Removing it.

Comment (nauar) [reply pickups_old.db:1]: This file does not appear anymore in the PR. Thanks for checking! :)

File: model.py [hunk 2]
```diff
@@ -29,7 +27,8 @@ class Meta:
 
 class Servers(Model):
     serverName = CharField(unique=True)
-    serverIp = CharField(unique=True)
+    serverIPv4 = CharField(unique=True, null=True)
```
[the remark this answers is not in the document]
Comment (nauar) [reply model.py:30]: Changed any reference to this field from `serverIp` to `serverIPv4`. I will implement the rest of changes in the issue #26 .

File: README.md [hunk 3]
```diff
@@ -14,28 +14,12 @@ A pickupbot for managing Xonotic pickup games
...
+```bash
 pip install -r requirements.txt
 ```
 
-Create database (**pickups.db**)
-```python
-python setupdb.py --createdb
-```
-
-*Optional*: import gametypes from gametypes.json
```
[the remark this answers is not in the document]
Comment (nauar) [reply README.md:26]: Noted, thanks!

**Review by Seekfried — APPROVED** [review]
Looks super!

**Pull request merged**